packages/codemods/tsup.config.ts

```diff
@@ -0,0 +1,17 @@
+import { defineConfig } from 'tsup'
+
+export default defineConfig([
+  {
+    format: 'cjs',
+    target: ['node18'],
```

Comment:
CLI runs on Node and currently targets the minimum LTS version, Node 18, as Node 16 LTS has ended.